pennylane/qnn/keras.py

```diff
@@ -52,6 +54,10 @@ class KerasLayer(Layer):
             arguments of the `add_weight()
             <https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer#add_weight>`__
             method and values being the corresponding specification.
+        argnum (Union[Sequence[int], int]): Argument location of the non-trainable inputs for
+            the circuit. This allows batch execution by creating executable circuits for each
+             input example with the same trainable weights. Default None.
+             See `pennylane.transforms.batch_input` for more details.
```

Comment:
```suggestion
        argnum (Union[Sequence[int], int]): Argument location of the non-trainable inputs for
            the circuit. This allows batch execution by creating executable circuits for each
            input example with the same trainable weights. Default ``None``.
            See :class:`~.pennylane.transforms.batch_input` for more details.
```

Ah, here is where the issue arises --- lines 2 and 3 don't match the indentation of line 1! I've also made a quick fix to the ReST markup to ensure that the link to `batch_input` is properly resolved.